### ISSUE TITLE
ValueError: Unrecognized configuration class <class 'transformers.models.mobilevit.configuration_mobilevit.MobileViTConfig'> for this kind of AutoModel: AutoModelForImageSegmentation.

### ISSUE THREAD

**Issue by xenova** [huggingface/optimum#1127]
### System Info

```shell
- `optimum` version: 1.8.8.dev0
- `transformers` version: 4.30.2
- Platform: Linux-5.15.107+-x86_64-with-glibc2.31
- Python version: 3.10.12
- Huggingface_hub version: 0.15.1
- PyTorch version (GPU?): 2.0.1+cu118 (cuda availabe: False)
- Tensorflow version (GPU?): not installed (cuda availabe: NA)
```

### Who can help?

@michaelbenayoun 

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [X] My own task or dataset (give details below)

### Reproduction

Running `optimum-cli export onnx -m apple/deeplabv3-mobilevit-small out`

results in:
```
Framework not specified. Using pt to export to ONNX.
Traceback (most recent call last):
  File "/usr/local/bin/optimum-cli", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/optimum/commands/optimum_cli.py", line 163, in main
    service.run()
  File "/usr/local/lib/python3.10/dist-packages/optimum/commands/export/onnx.py", line 219, in run
    main_export(
  File "/usr/local/lib/python3.10/dist-packages/optimum/exporters/onnx/__main__.py", line 184, in main_export
    model = TasksManager.get_model_from_task(
  File "/usr/local/lib/python3.10/dist-packages/optimum/exporters/tasks.py", line 1470, in get_model_from_task
    model = model_class.from_pretrained(model_name_or_path, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/transformers/models/auto/auto_factory.py", line 487, in from_pretrained
    raise ValueError(
ValueError: Unrecognized configuration class <class 'transformers.models.mobilevit.configuration_mobilevit.MobileViTConfig'> for this kind of AutoModel: AutoModelForImageSegmentation.
Model type should be one of DetrConfig.
```

### Expected behavior

The model should export correctly.

**Comment by regisss** [comment]
@xenova Should be fixed by #1128

**Comment by xenova** [comment]
Wow thanks for the quick response! 🚀
